src/components/sectionList/SectionListRow.tsx

```diff
@@ -32,10 +37,11 @@ export function SectionListRow<Model extends IdentifiableObject>({
             </DataTableCell>
             {selectedColumns.map(({ modelPropertyName }) => (
                 <DataTableCell key={modelPropertyName}>
-                    {/* TODO: Handle constant translations and resolve displayvalues to components */}
-                    {typeof modelData[modelPropertyName] === 'object'
-                        ? modelPropertyName
-                        : modelData[modelPropertyName]}
+                    {modelData[modelPropertyName] &&
+                        renderValue(
+                            modelPropertyName,
+                            modelData[modelPropertyName]
+                        )}
```

Comment:
You could argue we could render `ModelValue` directly here. However `ModelValue` depends on `Schema` - and there are sections that do not have a matching schema (eg. locales). So I decided to use a render-prop as to make this more composable.